Packaging/apple/Info.plist

```diff
@@ -48,7 +48,31 @@
 	<true/>
 	<key>UIFileSharingEnabled</key>
 	<true/>
+	<key>UIDeviceFamily</key>
+	<array>
+		<integer>1</integer>
+		<integer>2</integer>
+	</array>
 	<key>CADisableMinimumFrameDurationOnPhone</key>
 	<true/>
+	<key>UIRequiresFullScreen</key>
+	<true/>
+	<key>UISupportedInterfaceOrientations</key>
+	<array>
+		<string>UIInterfaceOrientationLandscapeLeft</string>
+		<string>UIInterfaceOrientationLandscapeRight</string>
+	</array>
+	<key>UISupportedInterfaceOrientations~ipad</key>
+	<array>
+		<string>UIInterfaceOrientationLandscapeLeft</string>
+		<string>UIInterfaceOrientationLandscapeRight</string>
+	</array>
+	<key>UISupportedInterfaceOrientations~iphone</key>
+	<array>
+		<string>UIInterfaceOrientationLandscapeLeft</string>
+		<string>UIInterfaceOrientationLandscapeRight</string>
+	</array>
```

Comment:
```suggestion
```
This is all handled already by SDL_SetHint https://github.com/diasurgical/devilutionX/blob/98cd3613a840b93ea2aaf78fc12bfebd231a72ce/Source/utils/display.cpp#L178

Reply:
I think it should be set in Info.plist as well.

Reply:
I think redundant settings are a waste of lines. But it's up to @AJenbo really.

Reply:
If it is not listed in Info.plist, it will always start with UIInterfaceOrientationLandscapeRight.

Reply:
Oh well, that's a valid reason, carry on. 👍

Reply:
that might fix some of the startup flickering